src/beta/resources/referral.js

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable no-dupe-class-members,no-unused-vars,class-methods-use-this */
+import superagent from 'superagent';
+
+import T from 'proptypes';
+import base from '../../resources/base';
+import { propTypes as userPropTypes } from '../../resources/user';
+import RequestError from '../../errors/request';
+import API from '../../easypost';
+
+export const propTypes = Object.assign({}, userPropTypes, {
+  name: T.string.isRequired, // mark as required
+  email: T.string.isRequired, // mark as required
+  phone: T.string.isRequired, // handle discrepancy between phone (create Referral) and phone_number (all User functions, retrieve Referral)
```

Comment:
remove the comment and rename `phone` to `phone_number`, the API endpoint is returning `phone_number` now